e2e_tests/tests/experiment/experiment.py

```diff
@@ -615,3 +617,21 @@ def s3_checkpoint_config_no_creds() -> Dict[str, str]:
 
 def root_user_home_bind_mount() -> Dict[str, str]:
     return {"host_path": "/tmp", "container_path": "/root"}
+
+
+def _export_and_load_model(experiment_id: int, master_url: str) -> None:
+    experimental.Determined(master_url).get_experiment(experiment_id).top_checkpoint().load()
+
+
+def export_and_load_model(experiment_id: int) -> None:
```

Comment:
please put a comment why this is running things in a subprocess.  In the past, we've done this for not affecting random seed too (not a good reason) so this is a good opportunity to explain what this tool is actually for.

Alternatively, you could just rename this "do_model_export_test", since you have that in a comment everywhere that you call it.

Reply:
done